config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  devise_for :users
+  root 'splash#index'
+  resources :group
+  resources :entities
```

Comment:
- [ ] I think it is better if you nest the `groups` and `entities` inside the `users` route. And if you further nest `entities` route inside `group` route it will be more meaningful from a REST point of view. :pray: